olm-catalog/serverless-operator/index/Dockerfile

```diff
@@ -6,7 +6,7 @@ COPY --from=quay.io/operator-framework/opm:latest /bin/opm /bin/opm
 COPY configs /configs
 
 RUN /bin/opm init serverless-operator --default-channel=stable --output yaml >> /configs/index.yaml
-RUN /bin/opm --skip-tls-verify render -o yaml registry.ci.openshift.org/knative/openshift-serverless-v1.22.0:serverless-stop-bundle \
+RUN /bin/opm --skip-tls-verify render -o yaml registry.ci.openshift.org/knative/openshift-serverless-v1.23.0:serverless-stop-bundle \
```

Comment:
Right now the image is not built, @mgencur should not this be available due to the release cut?

Reply:
The image will be available once any post-submit job runs (after any PR against 1.23 is merged).